curriculum/challenges/english/25-front-end-development/lecture-working-with-css-transforms-overflow-and-filters/672bccdb8f1823d8c60f914c.md

```diff
@@ -8,7 +8,40 @@ dashedName: what-is-a-css-reset
 
 # --description--
 
-Watch the lecture video and answer the questions below.
+Watch the video or read the transcript and answer the questions below.
+
+# --transcript--
+
+What is a CSS reset, and what are some common examples?
+
+A CSS reset is a stylesheet that removes all or some of the default formatting that web browsers apply to HTML elements. For example, you might have noticed that elements like paragraphs and headings already have margings by default even if you don't define them explicitly in you custom stylesheet. 
+
+You will also see this with various properties in a wide range of HTML elements. Different browsers may also apply different default styles to HTML elements. The default styles in Google Chrome may not be exactly the same as in Mozilla Firefox or Microsoft Edge. This can result in consistent styles across browsers which you should avoid as much as possible.
+
+To handle this, a CSS reset removes the default styles. By removing all or some of the default styles, you can get a consistent baseline for your design and minimize the potential inconsistencies across browsers and devices. Removing default styles can also make the styling process easier because you will only see the styles that you have applied expplicitly in your custom stylesheets. There are two main approaches to CSS resets: you can either define custom CSS resets or use third-party CSS resets.
+
+Custom CSS resets are stylesheets that you create from scratch to fit the need of your project. This way you can control the specific styles that will be reset with alot of room for flexibility. However, you also need to invest time to develop and maintain the stylesheets. Here's an example of a very common CSS rule for resetting the margin and padding of all HTML elements:
+
+```css
+* {
+  margin: 0;
+  padding: 0;
+}
+```
+
+It's usually written at the top of the CSS stylesheet. The asterisk star selector is a wildcard selector that matches all HTML elements, so they will have a default margin of zero and a default padding of zero on all four sides. This will give you a starting point and then you can customize them in more specific CSS selectors further down in the stylesheet.
+
+You can use this approach to select any HTML element and reset its default styles, just create a custom stylesheet and use the appropriate CSS selectors to match the elements and set the styles. 
+
+But this can be a time intensive process if you want to save time you can also use a third-party CSS reset. These stylesheets are already pre-built so you can just download them and add them too your project directly. A great example of a third-party CSS reset is `Normalize.css`. This stylesheet normalizes styles for a wide range of HTML elements  while still keeping some useful default styles especially those that are important for accessibility. It also corrects bugs and style inconsistencies.
```

Comment:
```suggestion
But this can be a time intensive process if you want to save time you can also use a third-party CSS reset. These stylesheets are already pre-built so you can just download them and add them to your project directly. A great example of a third-party CSS reset is `Normalize.css`. This stylesheet normalizes styles for a wide range of HTML elements  while still keeping some useful default styles especially those that are important for accessibility. It also corrects bugs and style inconsistencies.
```